src/Markdig/Renderers/Roundtrip/CodeBlockRenderer.cs

```diff
@@ -2,6 +2,7 @@
 // This file is licensed under the BSD-Clause 2 license. 
 // See the license.txt file in the project root for more information.
 
+using Markdig.Renderers.Normalize;
```

Comment:
Do you know why this namespace is added?

Reply:
Oh, maybe it is for XML documentation. I haven't looked at the code, but I'm curious what dependency we have with normalize...

Reply:
It was for the XML docs. The issue was that the comment was referencing the NormalizeRenderer (as that was used as a base for the roundtrip). I removed these usings and corrected the docs to point to the RoundtripRenderer.